### PR TITLE
Clang formatting

### PR DESCRIPTION
**Pull request by JohanLaine**

### CODE REVIEW

File: arduino/main/lib/src/SerialCommunication.cpp [hunk 1]
```diff
@@ -1,207 +1,197 @@
...
-#define ERROR_DATA_SIZE		12		// Package content: Invalid data size for the current command
-#define ERROR_OUT_OF_BOUND	13		// PAckage content: Value too large
-
-
-int SerialCommunication::readBuffer(byte * buffer, int bufferSize){
-	return Serial.readBytesUntil(PACKAGE_EOL, buffer, bufferSize);
+#define ERROR_PKG_SIZE 1   // Validation: Package too small
+#define ERROR_PKG_HEADER 2 // Validation: Invalid package header
+#define ERROR_PKG_CHKSUM 3 // Validation: Invalid checksum
+#define ERROR_UNKN_CMD 11  // Package content: Undefined command
```
Comment (JohanLaine) [inline arduino/main/lib/src/SerialCommunication.cpp:26]: Not sure if you like this better. I think it's easier right now to just stick to clang-format.

**Review by JohanLaine — COMMENTED** [review]
I installed clang-format-3.3 and the Atom plugin clang-format (version 1.30.0) made by LiquidHelium. https://atom.io/packages/clang-format

File: arduino/main/lib/src/MotorController.cpp [hunk 2]
```diff
@@ -2,61 +2,52 @@
...
-    analogWrite(_input_pin_2, duty);
+  enable_engine();
+  analogWrite(_input_pin_1, LOW);
+  analogWrite(_input_pin_2, duty);
 }
 
-void MotorController::disable_engine() {
-    digitalWrite(_enable_pin, LOW);
-}
+void MotorController::disable_engine() { digitalWrite(_enable_pin, LOW); }
```
Comment (ahemberg) [inline arduino/main/lib/src/MotorController.cpp:51]: I don't like this

**Review by ahemberg — APPROVED** [review]
I don't like the oneliners. Apart from that it's 'much better.

File: arduino/main/lib/src/MotorController.cpp (hunk 2 above)
Comment (JohanLaine) [reply arduino/main/lib/src/MotorController.cpp:51]: Typiskt italienare

**Pull request merged**